.jenkins/run_test.sh

```diff
@@ -0,0 +1,16 @@
+set -x
+
+pip install awscli pytest-html
+python .jenkins/inject_aws_credentials.py
```

Comment:
i'm not entirely sure that you need to do this...  the AWS login creds are passed in through env vars and should be available to the aws-cli commands later in the script.